Update to a cleaner logging output.

cdrage opened the pull request:
This updates the logging so it is much cleaner in it's output. 

Action/Mode Selected Is output has been updated to "Mode selected:" and
each run of Atomic App will also output both the version number of
Atomic App as well as the spec that's being used.

cdrage commented:
#dotests

cdrage commented:
Here's a screenshottt:

![img](http://i.imgur.com/pYCxwFJ.png)

cdrage commented:
ping @kadel @rtnpro @surajssd if you guys want to test / comment. This PR is meant to clean up loging UI/UX quite a bit. No more showing the `blahblah.py` file and such (unless on debug mode)

kadel commented:
:+1:  Looks nice and clean
LGTM

cdrage commented:
I have updated it again so answers asking are in-line with it as well ^^

cdrage commented:
I updated this PR yet again to include ASCII art, aligned logs as well as the answers.conf answering. Here's an example of what it looks like:

```
INFO   ::     _  _             _      _
INFO   ::    /_\| |_ ___ _ __ (_)__  /_\  _ __ _ __   Version:  0.5.1
INFO   ::   / _ \  _/ _ \ '  \| / _|/ _ \| '_ \ '_ \  Nulecule: 0.0.2
INFO   ::  /_/ \_\__\___/_|_|_|_\__/_/ \_\ .__/ .__/  Mode:     Run
INFO   ::                                |_|  |_|
INFO   :: Unpacking image: projectatomic/etherpad-centos7-atomicapp to /var/lib/atomicapp/projectatomic-etherpad-centos7-atomicapp-f5826f639b6a
INFO   :: Skipping pulling Docker image: projectatomic/etherpad-centos7-atomicapp
INFO   :: Extracting nulecule data from image: projectatomic/etherpad-centos7-atomicapp to /var/lib/atomicapp/projectatomic-etherpad-centos7-atomicapp-f5826f639b6a
INFO   :: Pulling external application for mariadb-centos7-atomicapp.
INFO   :: Unpacking image: projectatomic/mariadb-centos7-atomicapp to /var/lib/atomicapp/projectatomic-etherpad-centos7-atomicapp-f5826f639b6a/external/mariadb-centos7-atomicapp
INFO   :: Skipping pulling Docker image: projectatomic/mariadb-centos7-atomicapp
INFO   :: Extracting nulecule data from image: projectatomic/mariadb-centos7-atomicapp to /var/lib/atomicapp/projectatomic-etherpad-centos7-atomicapp-f5826f639b6a/external/mariadb-centos7-atomicapp
INFO   :: Provider not specified, using default provider - kubernetes
ANSWER >> db_user (Database User):
```

And a screenshot:
![img](http://i.imgur.com/iK4M4KF.png)

cdrage commented:
LOOK HOW PRETTY IT IS :sparkles: 
```
▶ sudo atomicapp run projectatomic/helloapache --destination helloworld --provider docker
INFO   ::     _  _             _      _
INFO   ::    /_\| |_ ___ _ __ (_)__  /_\  _ __ _ __   Version:  0.5.1
INFO   ::   / _ \  _/ _ \ '  \| / _|/ _ \| '_ \ '_ \  Nulecule: 0.0.2
INFO   ::  /_/ \_\__\___/_|_|_|_\__/_/ \_\ .__/ .__/  Mode:     Run
INFO   ::                                |_|  |_|
INFO   :: Unpacking image: projectatomic/helloapache to helloworld
INFO   :: Skipping pulling Docker image: projectatomic/helloapache
INFO   :: Extracting nulecule data from image: projectatomic/helloapache to helloworld
INFO   :: App exists locally and no update requested
INFO   :: Deploying to provider: Docker

Your application resides in helloworld
Please use this directory for managing your application
```

surajssd commented:
@cdrage cool, would look even better without `INFO    ::` which got printed alongwith the ASCII art.

cdrage commented:
I think we should keep it consistent with the INFO :)

rtnpro commented:
LGTM :+1:

cdrage commented:
Just a quick update. Need one more review @rtnpro @kadel of the code.

Updated to change WARNING to WARN due to the length of the word making the width of the logging a lot (word wrapping, etc.)

I also changed the formatting to add `-5s` to the logging output formatter, as that will automatically align the logging words / output.

cdrage commented:
@surajssd @containscafeine

Can any of you please review / thoughts for mergin' ? :)

kadel commented:
:+1: LGTM
